wcfsetup/install/files/lib/system/gridView/admin/BoxGridView.class.php

```diff
@@ -0,0 +1,148 @@
+ * @since       6.2
+ */
+final class BoxGridView extends AbstractGridView
+{
+    public function __construct()
+    {
+        $this->addColumns([
+            GridViewColumn::for('boxID')
+                ->label('wcf.global.objectID')
+                ->renderer(new ObjectIdColumnRenderer())
+                ->sortable(),
+            GridViewColumn::for('name')
+                ->label('wcf.global.name')
+                ->titleColumn()
+                ->filter(new TextFilter())
+                ->sortable(),
+            GridViewColumn::for('title')
+                ->label('wcf.global.title')
+                ->filter($this->getBoxContentFilter('title'))
+                ->hidden(),
+            GridViewColumn::for('content')
+                ->label('wcf.acp.box.content')
+                ->filter($this->getBoxContentFilter('content'))
+                ->hidden(),
+            GridViewColumn::for('boxType')
+                ->label('wcf.acp.box.type')
+                ->filter(
+                    new SelectFilter(
+                        \array_combine(
+                            Box::$availableBoxTypes,
+                            \array_map(
+                                static fn(string $type) => 'wcf.acp.box.type.' . $type,
+                                Box::$availableBoxTypes
+                            )
+                        )
+                    )
+                )
+                ->sortable(),
+            GridViewColumn::for('position')
+                ->label('wcf.acp.box.position')
+                ->filter(
+                    new SelectFilter(
+                        \array_combine(
+                            Box::$availablePositions,
+                            \array_map(
+                                static fn(string $position) => 'wcf.acp.box.position.' . $position,
+                                Box::$availablePositions
+                            )
+                        )
+                    )
+                )
+                ->sortable(),
+            GridViewColumn::for('showOrder')
+                ->label('wcf.global.showOrder')
+                ->filter(new NumericFilter())
+                ->sortable(),
+            GridViewColumn::for('originIsSystem')
+                ->label('wcf.acp.box.originIsNotSystem')
+                ->filter(
+                    new class extends BooleanFilter {
+                        #[\Override]
+                        public function applyFilter(DatabaseObjectList $list, string $id, string $value): void
+                        {
+                            $columnName = $this->getDatabaseColumnName($list, $id);
+
+                            $list->getConditionBuilder()->add("{$columnName} = ?", [0]);
+                        }
+                    }
+                )
+                ->hidden(),
+        ]);
+
+        $interactionProvider = new BoxInteractions();
+        $this->setInteractionProvider($interactionProvider);
```

Comment:
```suggestion
        $provider = new BoxInteractions();
        $provider->addInteractions([
            new Divider(),
            new EditInteraction(BoxEditForm::class)
        ]);
        $this->setInteractionProvider($provider);
```
In line with other grid views.